src/Env.js

```diff
@@ -1,3 +1,5 @@
-var MIDDLE_TIER_URL = "https://cs4500-sp19-random1.herokuapp.com"
+var MIDDLE_TIER_URL = (process.env.NODE_ENV === 'development' ?
```

Comment:
If this doesn't work we should keep it out for now. I didn't get a chance to look into other ways of doing it so I'd rather have the hardcoded heroku url

Reply:
Tyler says it worked for him. So maybe it's just some weird error with me _specifically_. Wil have to look into it